sig-network/charter.md

```diff
@@ -34,16 +34,16 @@ The following topics fall under ownership of this SIG:
 - Network Policy
   - APIs for defining network policies (e.g. [NetworkPolicy], [AdminNetworkPolicy], [BaselineAdminNetworkPolicy])
 - Cluster DNS.
-- Integration points with networking implementations (e.g. CNI integration).
-- Container runtime interface (CRI) (With [sig-node]).
+- Integration points with networking implementations (e.g. [Container Network Interface (CNI)][CNI] integration).
+- [Container Runtime Interface (CRI)][CRI] (With [sig-node]).
 - Cloud provider network integrations (With [sig-cloud-provider]).
 
 #### Cross-cutting and Externally Facing Processes
 
 ### Out of scope
 
-- The CNI specification itself, which is maintained outside the Kubernetes project
-- Particular implementations of the CNI specification
+- The [CNI] specification itself, which is maintained outside the Kubernetes project
+- Particular implementations of the [CNI] specification
 - Particular implementations of the NetworkPolicy API
```

Comment:
Except the new reference impl...

Reply:
Yes, and also the next line was untrue considering ingress-nginx. Updated both.